test/tsconfig.json

```diff
@@ -44,6 +44,7 @@
         "./rules/indentRuleTests.ts",
         "./rules/interfaceNameRuleTests.ts",
         "./rules/jsdocFormatRuleTests.ts",
+        "./rules/keywordNamedVariablesRule.ts",
```

Comment:
AKAIK, `filesGlob` is a special field used by atom-typescript (and maybe other IDEs) but isn't actually used by TS. The IDEs just auto-populate the files field based off of `filesGlob`.